core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,14 +154,14 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
```

Comment:
The case where the call returns `None` is not being handled in this method right now. You could do the `println(...)` and output the error here when the call returns `None`.